README.md

```diff
@@ -1,2 +1,63 @@
+<!-- omit in toc -->
 # memory-qt
-Memory game in Qt
+
+- [Goal Of The Repository](#goal-of-the-repository)
+- [Description Of The Application](#description-of-the-application)
+- [Download Compiled Binaries](#download-compiled-binaries)
+- [How To Build](#how-to-build)
+- [Use Cases](#use-cases)
+- [Requirements](#requirements)
+- [Ideas for Improvement](#ideas-for-improvement)
+
+
+# Goal Of The Repository
+
+This repository describes the implementation of an open-source Memory game in Qt, applying principles of software architecture such as requirements gathering and object-oriented analysis and design.
+
+This project uses the [Qt Widgets? / Qt Quick? TBD] approach.
+
+[Use cases](#use-cases) and [requirements](#requirements) are kept in the documentation directory (`docs`).
+
+# Description Of The Application
+
+Memory (also known as Concentration) is a card game in which all cards are dealt face down, and two cards are turned face up each turn. The object of the game is to locate pairs of matching cards in as few moves as possible.
```

Comment:
```suggestion
Memory (also known as Concentration) is a card game in which all cards are dealt face down, and two cards are turned face up each turn. The objective of the game is to locate pairs of matching cards in as few moves as possible.
```

Reply:
I'm also thinking, usually Memory is played by 2+ players. Should we specify this here?

Reply:
Good remark. I was thinking that, for simplicity's sake, the first prototype might just be a single-player version of the game, but we should definitely start reasoning about the actual number of players as soon as possible.

Reply:
I referenced it in a new issue